app/src/shared/components/Button/index.jsx

```diff
@@ -62,7 +64,7 @@ const Button = ({
     >
         {children}
         {waiting && (
-            <Spinner color="white" containerClassname={styles.spinnerContainer} />
+            <Spinner color={(!outline && darkBgs.has(kind)) ? 'white' : 'gray'} containerClassname={styles.spinnerContainer} />
```

Comment:
Added a spinner a style to make it pop more against the white background

Reply:
Looks like we use both `grey` and `gray` in the codebase. One day we should switch to my personal preference of `grey` :)